content/en/blog/2021/contributor-celebration.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Announcing the Kubernetes Contributor Celebration 2021"
+linkTitle: "Contributor Celebration 2021"
+date: 2021-12-06
+---
+
+By [Debabrata Panigrahi](https://twitter.com/debaelopedev)
+
+# Contributor Celebration 2021
+
+It's that time of the year again, Yayy!!
+
+Like last year, this year also we are back with Kubernetes Contributor Celebration, the annual end of the year celebration, to recognize our achievements and have some fun! It’s a time for us to relax, chat and do something fun with your fellow contributors!
+
+### Registration
+To register, please fill out the [Registration Form](https://docs.google.com/forms/d/e/1FAIpQLSdRIboKulEPf7jVgRH3iTVwS4PEk6k0htcOs1eSG2OBmkxEjg/viewform).
+More deatils on [how to join](https://www.kubernetes.dev/events/kcc2021/how-to-join/) the event 
+
+### Events and Activities
+1. [**DevOps Party Games**](https://www.kubernetes.dev/events/kcc2021/activities/#most-extreme-kubernetes-challenge---devops-party-game)
+ "DevOps Party Games takes the idea of "online party games" and tilts it on its head by adding DevOps-inspired content to existing games, and then streams it live via Twitch for a worldwide audience to watch, comment, and hopefully be entertained. In addition, the hosts (Matt Stratton, Jeremy Meiss, and Dan Maher) will provide color commentary, much like a modern day Cotton McKnight and Pepper Brooks (announcers from Dodgeball)." There will be a Kubernetes slant for this edition of DevOps Party Games.
+
+2. **[The Great Cloud Native Bakeoff](https://www.youtube.com/watch?v=koT08purWDc)**
+    A virtual cooking event styled after the [Great British Bake-off](https://en.wikipedia.org/wiki/The_Great_British_Bake_Off) will be held during the Kubernetes celebration Event 
+
+    Contestants will have 90 minutes to cook a Kubernetes themed dessert. Anything goes, as long as the majority of preparation is done live.  The competition is open to all skill levels, from baking from a cake mix to grinding your own flour as long as you can theme it to Kubernetes.
+    Contestants will be responsible for their own recipes and acquiring their ingredients. If you need help with either of these, let us know! 
+    Contestants will be expected to join a video conference call 30 mins prior to kick-off, and should have a phone or ipad with a stand (or a patient friend ) to capture them preparing their dessert.
+    Our Host and Commentators will provide commentary on our contestants and will ask each contestant questions throughout the event.
```

Comment:
please cut these three lines, it's too much detail, and is covered better by the form header.